src/java.xml/share/classes/javax/xml/catalog/Util.java

```diff
@@ -267,4 +269,34 @@ static void validateFeatureInput(Feature f, String value) {
             Util.validateUrisSyntax(value.split(";"));
         }
     }
+
+    /**
+     * Returns the absolute form of the specified uri after resolving it against
+     * the base. Returns the uri as is if it's already absolute.
+     *
+     * @param base the base, that is the system id of the catalog within the
+     * Catalog implementation
+     * @param uri the specified uri
+     * @return Returns the absolute form of the specified uri
```

Comment:
"Returns" may be redundant. Anyway, can this `Util` class be `final` and have private constructor not to be instantiated?